testing/regress/ecl/key/assert.xml

```diff
@@ -15,6 +15,7 @@
 <Exception><Code>100000</Code><Filename>assert.ecl</Filename><Line>46</Line><Source>user</Source><Message>Assert (2 = 1) failed [val1 = 1]</Message></Exception>
 <Exception><Code>100000</Code><Filename>assert.ecl</Filename><Line>47</Line><Source>user</Source><Message>Abc7</Message></Exception>
 <Exception><Code>100000</Code><Filename>assert.ecl</Filename><Line>67</Line><Source>user</Source><Message>Assert (2 = 9) failed [COUNT(ds) = 9]</Message></Exception>
+<Exception><Code>100000</Code><Filename>assert.ecl</Filename><Line>71</Line><Source>user</Source><Message>Assert (&#226;&#130;&#172; = &#226;&#130;&#172;&#226;&#130;&#172;) failed [euro = U&apos;&#226;&#130;&#172;&#226;&#130;&#172;&apos;]</Message></Exception>
```

Comment:
I'm not sure this is correct.  It suggests that it is being treated as a latin-1 string when converted to xml.  It should really be &#x20ac; - otherwise you are encoding the bytes rather than the characters.  I suspect it may be opening a whole can of worms...

Reply:
I think that's a separate bug in how ecl run operates - as you say a can of worms, but a separate one to the one I am fixing here.

I will open a separate Jira for that.